src/resources/l10n/en.json

```diff
@@ -134,7 +134,11 @@
     "attributeDeviceInactivityPeriod": "Device Inactivity Period",
     "attributeProcessingCopyAttributes": "Processing: Copy Attributes",
     "attributeColor": "Color",
+    "attributeWidth": "Width",
+    "attributeOpacity": "Opacity (0.0 - 1.0)",
     "attributeWebLiveRouteLength": "Web: Live Route Length",
+    "attributeWebRouteWidth": "Web: Route Width",
+    "attributeWebRouteOpacity": "Web: Route Opacity (0.0 - 1.0)",
```

Comment:
1. Why do we need two sets of attributes?
2. Normally we don't include value range in the string like this.

Reply:
1. A misunderstanding on my part on the use of resource strings - on is a geofence attribute on is the server/user attribute for routes but I see now they can both use the same resource string.
2. Opacity to me doesn't have a clear scale as it gets expressed in different ways ( 0.0-1.0, 0-100, 0-255 ) so it seemed helpful to express the range to avoid guessing/confusion as users aren't developers

Reply:
Actually I think having these as separate strings is necessary as they are for separate attributes - one attached to geofences, the other to user/server for routes. I'm not sure there is a common string that would make sense in both cases.

Reply:
Why not? Something like "Map Line Opacity".